playbook-website/app/javascript/site_styles/docs/_markdown.scss

```
@@ -123,6 +123,33 @@
     font-style: italic;
     padding: $space_sm $space_md;
   }
+  .changelog-container blockquote {
+    position: relative;
+    font-style: normal;
+    padding: 0px 0px 0px calc(#{$space_md} - 1px);
+
+    li, p {
+      font-size: $text_medium;
+    }
+  }
+  .changelog-container blockquote::before {
+    content: '';
+    display: inline-block;
+    background: $border_light;
+    width: 7px;
+    height: 100%;
+    border-radius: 5px;
+    position: absolute;
+    left: 0;
+    top: 0;
+  }
+
+  .changelog-container h4:last-of-type {
+    border-top: 1px solid $border_light;
+    padding-top: $space_md;
+    font-weight: $regular;
+  }
+
```

Two suggestions:

1. Why not move this code into it's own file. This file controls ALL the markdown for the entire site. This override mixed into the middle makes understanding this file more difficult. Overrides should be separated to improve clarity.
2. Utilize SASS power. You are repeating selectors when nesting will save you time and keep this DRY.

```suggestion
.changelog-container {
  blockquote {
    position: relative;
    font-style: normal;
    padding: 0px 0px 0px calc(#{$space_md} - 1px);
    &::before {
      content: '';
      display: inline-block;
      background: $border_light;
      width: 7px;
      height: 100%;
      border-radius: 5px;
      position: absolute;
      left: 0;
      top: 0;
    }
    li, p {
      font-size: $text_medium;
    }
  }
  h4:last-of-type {
    border-top: 1px solid $border_light;
    padding-top: $space_md;
    font-weight: $regular;
  }
}
```
Again I suggest moving it from the middle.

---

moved styles to a separate file and nested selectors. thanks!